superset/commands/security/create.py

```diff
@@ -36,13 +36,10 @@ def __init__(self, data: dict[str, Any]):
         self._tables = self._properties.get("tables", [])
         self._roles = self._properties.get("roles", [])
 
+    @transaction()
```

Comment:
I'm not sure why RLS doesn't have custom exceptions like most of the other commands.